app/src/test/java/protect/card_locker/MainActivityTest.java

```diff
@@ -195,15 +195,17 @@ public void testGroups() {
         assertEquals("Alphabetical two", groupTabs.getTabAt(2).getText().toString());
```

Comment:
Again can't comment on the correct line but it looks like the MainActivity fails to start due to a "NotFoundException". Perhaps it's trying to show the group._id which isn't a string?

Reply:
Ohhh woow, thanks so much for this comment! There was still a group._id I forgot in MainActivity that was probably causing the bug. I pushed a fix and hopefully it will run better now!
Thanks again!